apps/fees-pay/ccpay-bubble-frontend/perftest.yaml

```diff
@@ -6,6 +6,14 @@ spec:
   releaseName: ccpay-bubble-frontend
   values:
     nodejs:
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '512Mi'
```

Comment:
Due to memory usage around 300mb, even 380mb, it might not scale down once it is higher than 2.

Reply:
Hi Rapolas, Paybubble has regular reports run against it and constant user activity, so the memory could vary, I would prefer we keep this at 512M if we can and see what happens with the perftest.